src/labels/CustomLabels.labels

```diff
@@ -6547,6 +6725,14 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <!-- {0} = custom label commonPaymentServices i.e. Payment Services -->
         <value>For more information, please see {0} troubleshooting documentation on the Power of Us Hub.</value>
     </labels>
+    <labels>
+        <fullName>geFieldsNotFoundMessage</fullName>
```

Comment:
@bdvorachek I think this label should apply to your message for form render story?